packages/client/hmi-client/src/model-representation/petrinet/petrinet-renderer.ts

```diff
@@ -500,4 +500,13 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 				return { x: node.x, y: node.y };
 		}
 	}
+
+	// Given a specific id, find top level id
+	resolveChildId(id: string) {
+		return id;
+	}
```

Comment:
Is this a placeholder for a future feature?

Reply:
Nope, this is because the data scheme for petrinet-renderer is always a single level, so any given id is already at the top level and resolves to itself.  I can add a comment to clarify.

Reply:
Thank you